skillmap/src/components/SkillGraphContainer.tsx

```diff
@@ -1,31 +1,106 @@
 import * as React from "react";
+import { connect } from 'react-redux';
 
+import { SkillMapState } from '../store/reducer';
 import { SkillGraph } from "./SkillGraph";
+import { SvgGraph, getGraph, PADDING, UNIT, MIN_HEIGHT, MIN_WIDTH } from '../lib/skillGraphUtils';
 
 /* eslint-disable import/no-unassigned-import, import/no-internal-modules */
 import '../styles/skillgraph.css'
 /* eslint-enable import/no-unassigned-import, import/no-internal-modules */
 
 interface SkillGraphContainerProps {
     maps: SkillMap[];
+    graphs: SvgGraph[];
     backgroundImageUrl: string;
+    graphSize: {
+        width: number;
+        height: number;
+    };
 }
 
-export class SkillGraphContainer extends React.Component<SkillGraphContainerProps> {
+interface SkillGraphContainerState {
+    backgroundSize: {
+        width: number;
+        height: number;
+    };
+}
+
+const THRESHOLD = 0.05;
+
+export class SkillGraphContainerImpl extends React.Component<SkillGraphContainerProps, SkillGraphContainerState> {
+    constructor(props: SkillGraphContainerProps) {
+        super(props);
+
+        this.state = { backgroundSize: { width: 0, height: 0 } };
+    }
+
+    protected onImageLoad = (evt: any) => {
+        this.setState({
+            backgroundSize: {
+                    width: evt.target.offsetWidth,
+                    height: evt.target.offsetHeight
+                }
+            })
```

Comment:
```suggestion
                width: evt.target.offsetWidth,
                height: evt.target.offsetHeight
            }
        })
```

nit: looks double indented